page.codeberg.foreverxml.Random.json

```diff
@@ -0,0 +1,83 @@
+{
+    "app-id" : "page.codeberg.foreverxml.Random",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "randomgtk",
+    "finish-args" : [
```

Comment:
Consider adding `--device=dri` to silence these warnings:

```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: MESA-LOADER: failed to open amdgpu: /usr/lib/x86_64-linux-gnu/GL/default/lib/dri/amdgpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/GL/default/lib/dri)
libGL error: failed to load driver: amdgpu
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: radeonsi
```

Reply:
I will do that. Thanks for the info :)